content/concepts/did-ddo.md

```diff
@@ -180,6 +180,7 @@ An asset should have at least one service to be actually accessible, and can hav
 | **`files`**            | [Files](#files)             | **/**                           | Encrypted file URLs.                                                                                                                         |
 | **`timeout`**          | `number`                    | **✓**                           | Describing how long the service can be used after consumption is initiated. A timeout of `0` represents no time limit. Expressed in seconds. |
 | **`privacy`**          | [Privacy](#compute-privacy) | **✓** (for compute assets only) | If service is of `type` `compute`, holds information about the compute-related privacy settings.                                             |
+| **`resources`**         | [Resources](#compute-resources) | **✓** (for compute assets only) | If service is of `type` `compute`, holds information about the compute-related resources.                                             |
```

Comment:
why have `privacy` & `resources` and not just one key for `compute` as both of them are only needed for compute? Simple flat structure then and immediately obvious it only applies to compute assets:

```
interface Compute {
  namespace: string
  cpu: number
  gpu: number
  gpuType: string
  memory: string
  volumeSize: string
  allowRawAlgorithm: boolean
  allowNetworkAccess: boolean
  publisherTrustedAlgorithmPublishers: string[]
  publisherTrustedAlgorithms: PublisherTrustedAlgorithm[]
}
```

Reply:
agreed